openedx/core/djangoapps/notifications/models.py

```diff
@@ -6,18 +6,52 @@
 from model_utils.models import TimeStampedModel
 from opaque_keys.edx.django.models import CourseKeyField
 
+NOTIFICATION_CHANNELS = ['web', 'push', 'email']
+
 # When notification preferences are updated, we need to update the CONFIG_VERSION.
 NOTIFICATION_PREFERENCE_CONFIG = {
-    "discussion": {
-        "new_post": {
-            "web": False,
-            "push": False,
-            "email": False,
+    'discussion': {
+        'enabled': False,
+        'core': {
+            'info': '',
+            'web': False,
+            'push': False,
+            'email': False,
+        },
+        'notification_types': {
+            'new_post': {
+                'info': '',
+                'web': False,
+                'push': False,
+                'email': False,
+            },
         },
+        'not_editable': {},
```

Comment:
Can you also add a comment here how this field will be populated and how it will work?